src/pages/Welcome/Welcome.css

```diff
@@ -1,93 +1,93 @@
-.welcome-main{
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    width: 100%;
-    height: 100%;
-    flex-direction: column;
+.welcome-main {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
 }
 
-.welcome-text{
-    font-size: 5rem;
+.welcome-text {
+  font-size: 5rem;
 }
 
-.input{
-    outline: none;
-    padding: .5rem 1rem;
-    font-size: 1.5rem;
-    text-align: center;
-    color: white;
-    margin-top: 1rem;
-    background: rgba( 255, 255, 255, 0.25 );
-    box-shadow: 0 8px 32px 0 rgba(250, 250, 250, 0.37);
-    backdrop-filter: blur( 8px );
-    border-radius: 10px;
-    border: 1px solid rgba( 255, 255, 255, 0.18 );
+.input {
+  outline: none;
+  padding: 0.5rem 1rem;
+  font-size: 1.5rem;
+  text-align: center;
+  color: white;
+  margin-top: 1rem;
+  background: rgba(255, 255, 255, 0.25);
```

Comment:
Suggestion : The colors are being used at multiple places you can use a variable for the same